models/models.js

```diff
@@ -17,3 +17,23 @@ exports.fetchArticles = () => {
       return articles;
     });
 };
+
+exports.fetchArticlesByID = (article_id) => {
+  const query = `SELECT * FROM articles WHERE articles.article_id =$1`;
+  return db.query(query, [article_id]).then(({ rows, rowCount }) => {
+    if (rowCount === 0) {
+      return Promise.reject({ status: 404, msg: "Article not found" });
+    }
+    return rows[0];
+  });
+};
+
+exports.fetchComments = (article_id) => {
+  const query = `SELECT * FROM comments WHERE article_id =$1 ORDER by created_at DESC`;
+  return db.query(query, [article_id]).then(({ rows, rowCount }) => {
+    if (rowCount === 0) {
+      return Promise.reject({ status: 404, msg: "Article not found" });
+    }
+    return rows[0];
```

Comment:
this will only ever return a single comment for an article ... the ticket wants an array of multiple comments